cli/flags.rs

```diff
@@ -198,46 +179,7 @@ fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
 }
 
 #[cfg_attr(feature = "cargo-clippy", allow(stutter))]
-pub fn set_flags(
-  args: Vec<String>,
-) -> Result<(DenoFlags, Vec<String>), String> {
-  let mut rest_argv: Vec<String> = vec!["deno".to_string()];
-  let cli_app = create_cli_app();
-  let matches = cli_app.get_matches_from(args);
-
-  match matches.subcommand() {
-    ("eval", Some(info_match)) => {
-      let code: &str = info_match.value_of("code").unwrap();
-      rest_argv.extend(vec![code.to_string()]);
-    }
-    ("info", Some(info_match)) => {
-      let file: &str = info_match.value_of("file").unwrap();
-      rest_argv.extend(vec![file.to_string()]);
-    }
-    ("fmt", Some(fmt_match)) => {
-      let files: Vec<String> = fmt_match
-        .values_of("files")
-        .unwrap()
-        .map(String::from)
-        .collect();
-      rest_argv.extend(files);
-    }
-    (script, Some(script_match)) => {
-      rest_argv.extend(vec![script.to_string()]);
-      // check if there are any extra arguments that should
-      // be passed to script
-      if script_match.is_present("") {
-        let script_args: Vec<String> = script_match
-          .values_of("")
-          .unwrap()
-          .map(String::from)
-          .collect();
-        rest_argv.extend(script_args);
-      }
-    }
-    _ => {}
-  }
-
+pub fn set_flags(matches: ArgMatches) -> Result<DenoFlags, String> {
```

Comment:
Regarding the return value - it doesn't seem like it ever errors? Can this be just
```rust
pub fn set_flags(matches: ArgMatches) -> DenoFlags
```

Reply:
Yep, will do

Reply:
Can you also rename this function into something like "parse_flags" ?